docs/docs/Best_Practices/Sensitive_Data.md

```diff
@@ -34,6 +34,9 @@ This tells the Delphix Engine to take special precautions with this password pro
 3. The Delphix Engine's UI and CLI will not display the password.
 4. Clients of the Delphix Engine's public API will not be able to access the password.
 
+!!! note
+  Removing a previously added password property from a field and running a [Data Migration](Glossary.md#data-migration) will expose the password in plaintext.
```

Comment:
Do we want to leave it as a note or add some recommendation - for e.g. "please make sure such properties handled as such in the new schema unless the intent is to change the property type". something like that?

Reply:
Maybe something like "If a new release of your plugin removes the `password` keyword from a previously-existing property, then the Delphix engine will stop taking these precautions, even with already-exiting data. So, for example, the property will no longer be stored in encrypted form."

Reply:
I'll add another sentence to tell the plugin author to write a migration for the property change, but I think it's already clear that their password will not be encrypted after their change.